src/avt/Math/CMakeLists.txt

```diff
@@ -7,34 +7,30 @@
 #
 #****************************************************************************
 
-SET(AVTMATH_SOURCES
-avtFactorial.C
-avtHermitePolynomial.C
-avtMatrix.C
-avtPlane.C
-avtPolynomial.C
-avtQuaternion.C
-avtTrackball.C
-avtVecArray.C
-avtVector.C
-)
-
-INCLUDE_DIRECTORIES(
-${CMAKE_CURRENT_SOURCE_DIR}
-${VISIT_COMMON_INCLUDES}
-${VTK_INCLUDE_DIRS}
-)
-
 if(VTK_VERSION VERSION_EQUAL "8.1.0")
     set(vtklibs vtkCommonCore)
 else()
     set(vtklibs VTK::CommonCore)
 endif()
 
-# Add link directories
-LINK_DIRECTORIES(${LIBRARY_OUTPUT_DIRECTORY} )
+blt_add_library(
+    NAME       avtmath
```

Comment:
Are these missing pre-fixing by `$(CMAKE_CURRENT_SOURCE_DIR}/` as in `src/common/Exceptions/Database/CMakeLists.txt`

Reply:
`CMAKE_CURRENT_SOURCE_DIR` isn't needed here because the files are being processed in this this directory, and if there is no path, or even relative path, CMake assumes CWD.
`CMAKE_CURRENT_SOURCE_DIR` is needed in src/common/XXX/CMakeLists.txt because the files are being appended to a list and processed in a different dir (/src/common/CMakeLists.txt).